### PR TITLE
Remove language_css_alter() (RTL stylesheets) in favor of HTML 'dir' attribute.

### PR DESCRIPTION
**Pull request by adamtothemoon**
pr for: https://github.com/backdrop/backdrop-issues/issues/207
For some reason the .rej files appear in all branches not just the one I was working on. Any idea why that would be?

### CODE REVIEW

**Comment by jenlampton** [comment]
.rej files will appear (everywhere, as untracked files) until you delete them. Make sure you apply all the changes within first though!

File: core/misc/dropbutton/dropbutton.theme-rtl.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ */
+
+.dropbutton-multiple .dropbutton {
+  border-left: 1px solid #e8e8e8;
+  border-right: 0 none;
+}
+.dropbutton-multiple .dropbutton li > * {
+  margin-left: 0.25em;
+  margin-right: 0;
+}
```
Comment (jenlampton) [inline core/misc/dropbutton/dropbutton.theme-rtl.css]: I don't think this is correct. This patch should be removing all the -rtl.css files, and adding that code into the non-rtl versions.

**Comment by jenlampton** [comment]
Closing because this needs work.  Please re-open when you have a new PR for review.

**Comment by adamtothemoon** [comment]
pr for https://github.com/backdrop/backdrop-issues/issues/207

**Comment by jenlampton** [comment]
Looks like @adamtothemoon does not have permission to re-open his own pull request.

File: core/misc/vertical-tabs.css [hunk 2]
```diff
@@ -29,7 +44,7 @@ fieldset.vertical-tabs-pane > legend {
   margin: 0;
 }
 .vertical-tabs .vertical-tabs-list a {
-  display: block;
+  display: block;gi
```
Comment (quicksketch) [inline core/misc/vertical-tabs.css:47]: Looks like a typo here.

File: core/misc/dropbutton.css [hunk 3]
```diff
@@ -132,3 +132,19 @@
...
+ */
+
+[dir="rtl"] .dropbutton-multiple .dropbutton {
+  border-left: 1px solid #e8e8e8;
+  border-right: 0 none;
+}
+[dir="rtl"] .dropbutton-multiple .dropbutton li > * {
+  margin-left: 0.25em;
+  margin-right: 0;
+}
```
Comment (quicksketch) [inline core/misc/dropbutton.css:150]: We don't need to add the \@file docblock here, and these selectors should be next to the matching rules.

File: core/themes/seven/vertical-tabs.css [hunk 4]
```diff
@@ -69,9 +82,28 @@ fieldset.vertical-tabs-pane {
...
+[dir="rtl"] .vertical-tabs .selected a,
+[dir="rtl"] .vertical-tabs .selected a:hover,
+[dir="rtl"] .vertical-tabs .selected a:focus,
+[dir="rtl"] .vertical-tabs .selected a:active {
+  border-left-color: #fff;
+
+
+
+
+
```
Comment (quicksketch) [inline core/themes/seven/vertical-tabs.css:109]: ~~Hmm, looks like we're missing the closing } bracket here.~~

Actually looks like this entire last hunk can be deleted, as it's changes are included already above.

File: core/modules/color/color.admin.css [hunk 5]
```diff
@@ -21,39 +24,67 @@
...
+[dir=rtl] #palette .up {
+  background-position: 0 -27px;
+}
 #palette .both {
   background-position: 100% -54px; /* LTR */
 }
-
+[dir=rtl] #palette .both {
+  background-position: 0 -54px;
+}s
```
Comment (quicksketch) [inline core/modules/color/color.admin.css:87]: Trailing "s" looks like a typo here.

**Comment by quicksketch** [comment]
Looks like there is a bunch of RTL sheets still unconverted in this PR. I've filed a replacement at https://github.com/backdrop/backdrop/pull/334 that includes all these changes plus the missing RTL stylesheets.

**Pull request closed without merging**